src/components/ResourceBrowser/ResourceList/ResourceBrowserActionMenu.tsx

```diff
@@ -102,6 +120,18 @@ export default function ResourceBrowserActionMenu({
                     removeTabByIdentifier={removeTabByIdentifier}
                 />
             )}
+
+            {showVulnerabilityModal && ScanResourceModal && (
+                <ScanResourceModal
```

Comment:
pls check if both the modals won't open together

Reply:
Checked